detox/package.json

```diff
@@ -38,6 +38,7 @@
     "wtfnode": "^0.8.0"
   },
   "dependencies": {
+    "nodejs-base64": "^1.0.3",
```

Comment:
I would like to discourage the usage of an external library for base64 encoding/decoding. It is available in vanilla node:
`return Buffer.from(str, 'base64').toString('utf8');`

Moreover, this specific library is only 34 lines of code, which basically just runs the line I pasted above.

Reply:
awesome, switching-to!

Reply:
pushed a new commit